tests/unit/copying_manager_test.py

```diff
@@ -256,41 +289,6 @@ def test_update_config_not_monitored(self):
         matchers = self._manager.log_matchers
         self.assertEquals(0, len(matchers))
 
-    def test_schedule_log_path_for_removal_and_re_add_before_actual_removal(self):
```

Comment:
Did this test case get moved somewhere else?

Reply:
I've got it back. I've missed it somehow, thanks.